CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## Unreleased
 
-* Drop support for Ruby 3.0. The minimum required Ruby version is now 3.1
+* **BREAKING CHANGE** Drop support for Ruby 3.0. The minimum required Ruby
+  version is now 3.1
```

Comment:
```suggestion
* Drop support for Ruby 3.0. The minimum required Ruby
  version is now 3.1
```

We don't have flag dropping Ruby versions as breaking changes: https://docs.publishing.service.gov.uk/manual/publishing-a-ruby-gem.html#ruby-version-compatibility

Reply:
Thank you for reviewing. I've removed the "breaking change" label from the line related to Ruby update and will merge it soon and then create the major version release PR.